Add suggestion for IIFEs to avoid with statements (#26020)
mdn/content: files/en-us/web/javascript/reference/statements/with/index.md

}
```

### Avoiding the with statement by using an IIFE

> If you're producing an expression that must reuse a long-named reference multiple times, and your goal is to eliminate that lengthy name within your expression, you may prefer to provide the long-named reference as an argument to an [IIFE](/en-US/docs/Glossary/IIFE).

Reviewer: ```suggestion
If you're producing an expression that must reuse a long-named reference multiple times, and your goal is to eliminate that lengthy name within your expression, you can wrap the expression in an [IIFE](/en-US/docs/Glossary/IIFE) and provide the long name as an argument.
```
